golang-regex-validator/regexCompiler.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"io"
+	"os"
+	"regexp"
+)
+
+func main() {
+	bytes, err := io.ReadAll(os.Stdin)
+	if err != nil {
+		panic(err)
+	}
+	regex := string(bytes)
+
+	regexp.MustCompile(regex)
```

Comment:
It seems that when the compilation fails, the exception will always contain the whole regular expression. The regular expression combining all the message filters could be a rather long one, making a test job output difficult to navigate. It is probably fine, and I do not expect to see frequent failures here. Just wanted to point it out.

Reply:
You are right. I prefer this approach as we do not need to capture the error and then put some custom message. But if that will be your preference, should be easy to do, just couple of lines. Let me know your preference. The issue with a custom message is that we might miss some useful information the error message might contain. Also having the whole regex printed out might be good for troubleshooting, in case you believe the input is something but it is actually something else (some formatting or things like that).